include/bluetooth/hci.h

```diff
@@ -484,6 +484,11 @@ struct bt_hci_write_local_name {
 #define BT_BREDR_SCAN_INQUIRY                   0x01
 #define BT_BREDR_SCAN_PAGE                      0x02
 
+#define BT_HCI_OP_WRITE_CLASS_OF_DEVICE        BT_OP(BT_OGF_BASEBAND, 0x0024)
```

Comment:
minor: please indent the BT_OP... identically to the rest of this file. It seems you're missing one more space before it.

Reply:
Updated, thanks.